webrtc.html

```diff
@@ -3273,70 +3273,72 @@ <h2>Callback <a class="idlType">RTCSessionDescriptionCallback</a>
         </section>
       <section>
         <h4>Legacy configuration extensions</h4>
-        <div>
-          <pre class="idl">partial dictionary RTCOfferOptions {
-            boolean offerToReceiveAudio;
-            boolean offerToReceiveVideo;
-          };
-          </pre>
-          <section>
-            <h2>Attributes</h2>
-            <dl data-link-for="RTCOfferOptions" data-dfn-for="RTCOfferOptions" class="dictionary-members">
-              <dt><dfn>offerToReceiveAudio</dfn> of type <span class="idlMemberType"><a>boolean</a></span></dt>
-              <dd>
-                <p>Offer to receive audio even though no audio will be sent.
-                When this dictionary member is present as an option to <a href=
-                "#dom-rtcpeerconnection-createoffer">createOffer</a>, run the
-                <a>offer to receive</a> steps with a <var>mediaType</var> of
-                "audio" instead of the regular <a href=
-                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
-              </dd>
-              <dt><dfn>offerToReceiveVideo</dfn> of type <span class="idlMemberType"><a>boolean</a></span></dt>
-              <dd>
-                <p>Offer to receive video even though no video will be sent.
-                When this dictionary member is present as an option to <a href=
-                "#dom-rtcpeerconnection-createoffer">createOffer</a>, run the
-                <a>offer to receive</a> steps with a <var>mediaType</var> of
-                "video" instead of the regular <a href=
-                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
-              </dd>
-            </dl>
-            <p>To <dfn>offer to receive</dfn> with <var>mediaType</var>, run
-            the following steps:</p>
+        <p>When <a href="#dom-rtcpeerconnection-createoffer">createOffer</a>
+        is called with any of the legacy options specified in this section,
+        run the followings steps instead of the regular <a href=
+        "#dom-rtcpeerconnection-createoffer">createOffer</a> steps:</p>
+        <ol>
+          <li>
+            <p>Let <var>options</var> be the methods second argument.</p>
```

Comment:
first argument? (I am glad we're dealing with the legacy callbacks elsewhere)

Reply:
Will fix!